src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -257,3 +278,30 @@ void GraphingCalculator::OnZoomResetCommand(Object ^ /* parameter */)
 {
     GraphingControl->ResetGrid();
 }
+
+void GraphingCalculator::OnActiveTracingClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    GraphingControl->ActiveTracing = !GraphingControl->ActiveTracing;
+}
+
+void CalculatorApp::GraphingCalculator::OnGraphLostFocus(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    // If the graph is losing focus while we are in active tracing we need to turn it off so we don't try to eat keys in other controls.
+    if (GraphingControl->ActiveTracing)
+    {
+        GraphingControl->ActiveTracing = false;
+        OnShowTracePopupChanged(false);
+    }
+}
+
+void CalculatorApp::GraphingCalculator::OnLoosingFocus(Windows::UI::Xaml::UIElement ^ sender, Windows::UI::Xaml::Input::LosingFocusEventArgs ^ args)
+{
+    FrameworkElement ^ newFocusElement = (FrameworkElement ^) args->NewFocusedElement;
```

Comment:
Use static_cast instead

Reply:
Actually going with a dynamic_cast I can do similar to c# "as" functionality and if it comes back null it's not the control I care about.